### PR TITLE
[14.0][MIG] purchase_request_tier_validation

### PR DESCRIPTION
**Pull request by SurachetS**
depends on 
- [x] base_tier_validation

### CODE REVIEW

**Comment by kittiu** [comment]
base_tier_validation is merged https://github.com/OCA/server-ux/pull/226

File: purchase_request_tier_validation/README.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
+================================
```
Comment (kittiu) [inline purchase_request_tier_validation/README.rst]: You can delete this file (it is on v13 too).
When merged, README.rst will be generated automatically from readme/*.*

File: purchase_request_tier_validation/views/purchase_request_view.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                        name="tier_validated"
+                        string="Validated"
+                        domain="[('validated', '=', True)]"
+                        help="Purchase Requests validated and ready to be confirmed"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+</odoo>
```
Comment (kittiu) [inline purchase_request_tier_validation/views/purchase_request_view.xml:129]: I saw in this file, that the view `ext_id = tier_review_form_wiz ` was deleted.
What was the use of this view in v13? And why delete it?

@AdriaGForgeFlow do you know about this?

Comment (AdriaGForgeFlow) [reply purchase_request_tier_validation/views/purchase_request_view.xml:129]: I cannot see where it was used before being deleted. Probably it was not needed.

File: purchase_request_tier_validation/README.rst (hunk 1 above)
Comment (SurachetS) [reply purchase_request_tier_validation/README.rst]: ok

File: purchase_request_tier_validation/views/purchase_request_view.xml (hunk 2 above)
Comment (kittiu) [reply purchase_request_tier_validation/views/purchase_request_view.xml:129]: Thank you @AdriaGForgeFlow 
@SurachetS deletion is approved.

**Comment by kittiu** [comment]
@HviorForgeFlow Travis still red. I can't recon the error, is this current Travis problem (limit), Do you know?

**Comment by kittiu** [comment]
@SurachetS base_tier_validation is already merged. To make it work, you need to add `server-ux` to oca_dependencies.txt file (root of the repo), to let travis know what to include in addons path.

**Comment by SurachetS** [comment]
> @SurachetS base_tier_validation is already merged. To make it work, you need to add `server-ux` to oca_dependencies.txt file (root of the repo), to let travis know what to include in addons path.

thank you :)

File: purchase_request_tier_validation/views/purchase_request_view.xml (hunk 2 above)
Comment (SurachetS) [reply purchase_request_tier_validation/views/purchase_request_view.xml:129]: thank you

**Review by kittiu — APPROVED** [review]
LGTM

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Pull request merged**